docker-compose.yaml

```diff
@@ -22,22 +22,26 @@ x-common-environment: &common-environment
 
 x-content-publishing-env: &content-publishing-env
   START_PROCESS: content-publishing-api
-  FILE_UPLOAD_MAX_SIZE_IN_BYTES: 2000000000
+  FILE_UPLOAD_MAX_SIZE_IN_BYTES: 500000000
+  FILE_UPLOAD_COUNT_LIMIT: 10
   ASSET_EXPIRATION_INTERVAL_SECONDS: 300
   BATCH_INTERVAL_SECONDS: 12
   BATCH_MAX_COUNT: 1000
   ASSET_UPLOAD_VERIFICATION_DELAY_SECONDS: 5
+  CACHE_KEY_PREFIX: 'content-publishing:'
```

Comment:
nit: is there one for account service ?

Reply:
`account-service` was already up to date. 🤷‍♂️